indico/htdocs/js/indico/modules/list_generator.js

```diff
@@ -60,7 +60,8 @@
                 0: {
                     sorter: false
                 }
-            }
+            },
+            sortReset: true
```

Comment:
more readable if you put it above the headers options (i.e. have the multiline dict ones last)